config/locales/fr/restroom.fr.yml

```diff
@@ -20,6 +20,9 @@ fr:
     guess_location: 'Trouver ma position actuelle'
     directions: 'Indications'
     directions_hint: "(ex: troisième étage au fond, à coté des cabines d'essayage...)"
+    distance:
+      miles: milles
+      kilometers: kilomètres
```

Comment:
Hello, @BertrandClosset, @TiteiKo.

Sorry to bother you... It has been a while since you contributed the French translation for Refuge Restrooms (#450).

We are trying to add proper translation of "miles" and "kilometers" to the website, and so I wonder if you could help by checking if this new translation is correct, or suggest a better translation?

Thank you very much.

Sincerely,

\- DeeDeeG

Reply:
Hi @DeeDeeG,

No problem, it's always a pleasure to help. In my opinion, there is nothing to change to this translation ;)

Bertrand

Reply:
Thank you!